tests/test_fields.py

```diff
@@ -1343,15 +1343,22 @@ def setup_class(cls):
         cls.field = serializers.DateTimeField()
         cls.kolkata = pytz.timezone('Asia/Kolkata')
 
+    @staticmethod
+    def is_utc(tz):
+        """Django 2.1 uses datetime.timezone.utc
+
+        Ref: https://github.com/django/django/pull/9484#issuecomment-399169042"""
+        return tz.utcoffset(None).total_seconds() == 0.0
```

Comment:
This check seems a bit funky. Is there no better way to do this?

Reply:
Agreed - I am also not happy with it.

Asked about in the commit making this necessary: https://github.com/django/django/pull/9484#issuecomment-399169042.

If Django does not change `get_default_timezone` itself we could also change the test to compare another (the new) "utc" (which is datetime.timezone.utc).

Reply:
In retrospect, I don't think it's incredibly important to block this PR over the test code. It could be improved, but it's not worth blocking.

Reply:
The behaviour here will be restored for Django v2.1b2. So if we can keep this in a separate commit it'll just be a revert at that point.

Reply:
> The behaviour here will be restored for Django v2.1b2

Did they revert the original PR?

Reply:
No code changes AFAIK.
The assumption might be because of https://code.djangoproject.com/ticket/29514 being accepted.